mobile/ios/ios-proxy-services.ts

```diff
@@ -301,6 +301,14 @@ export class NotificationProxyClient {
 		}
 	}
 
+	private postNotificationCore(notificationName: string): void {
+		this.plistService.sendMessage({
```

Comment:
No, it is not possible. When `postNotificationAndAttachForData` is called we initialize `plistService` inside `openSocket` function - https://github.com/telerik/mobile-cli-lib/pull/251/files#diff-b2b4fa0509e169323e6cd005ec98dc9bL234